georchestra-integration/pom.xml

```diff
@@ -12,13 +12,22 @@
   <name>geOrchestra integration</name>
   <properties>
     <rootProjectDir>${basedir}/..</rootProjectDir>
-    <georchestra.version>25.0.1-SNAPSHOT</georchestra.version>
+    <georchestra.version>24.0.11</georchestra.version>
```

Comment:
a bit concerned about that detail.. that means we cant use new features (in the commons jars etc) that would happen to be in the master branch ? in the past we've had branches per-georchestra-branch for that .. cf https://github.com/georchestra/geonetwork/branches & the `georchestra-gn4.2.x-2xxx` branches. that would also allow stable vs main/master. But i'm not sure that matters much, just thinking out loud what would happen if there were diverging stuff in the deps.

the PR's first comment could explain more what the goal is ;)

Reply:
sure, and i guess eventually branch this repo if some commits depend on something that isnt in the stable georchestra branch ?